src/pages/About/About.tsx

```diff
@@ -0,0 +1,320 @@
+import "./About.css";
+import {
+  handleLogin,
+  clientId,
+  scope,
+  redirectUrl,
+} from "../../utils/LoginFunctions";
+import {
+  dashOne,
+  dashTwo,
+  dashThree,
+  dashFour,
+  aboutStrava,
+  dashTable,
+  dashMap,
+  dashGraph,
+} from "../../assets/about";
+import { motion, useAnimation } from "framer-motion";
+import { useInView } from "react-intersection-observer";
+import { useEffect } from "react";
+import {
+  framerImageVariants,
+  framerTextVariants,
+  secondBlockChildren,
+  secondBlockVariants,
+} from "./utils/FramerVaraints";
+import { handleHover, handleHoverEnd } from "./utils/Functions";
+
+const About = () => {
```

Comment:
Try to break this component into smaller re-usable components seems like many parts use the same props.

Reply:
This part repeats it self quite many times you can create a smaller component and send some attributes as props.
`<div className="circle">20km</div>
        <div className="timeline--element">
          <div className="timeline--line--container">
            <div className="timeline--line"></div>
          </div>
          <div ref={blockFour} className="timeline--block">
            <motion.img
              className="timeline--image timeline--image--table"
              variants={framerImageVariants}
              initial="visible"
              animate={animationFour}
              src={dashMap}
            />
            <motion.div
              className="timeline--text"
              animate={textAnimationFour}
              variants={framerTextVariants}
              initial="visible"
            >
              <h2 className="timeline--title">See where you've been</h2>
              <p className="timeline--description">
                The map gives a view of all of your runs, centered on your most
                recent run.
              </p>
            </motion.div>
          </div>
        </div>`

Reply:
ill add this to my refactoring mission

Reply:
hey. I've done a job of breaking it down a bit. 

I'm not sure if you wanted me to go further with this?